src/assets/scripts/client/aircraft/FlightManagementSystem/LegModel.js

```diff
@@ -221,6 +221,10 @@ export default class LegModel {
         return this._waypointCollection;
     }
 
+    get altitude() {
+        return this._procedureModel.altitude;
```

Comment:
What happens when we run this on a leg that isn't a procedural leg? Like a direct leg?

Reply:
returns undefined.

Reply:
okay, and it looks like that is subsequently handled well in `RouteModel.getInitialClimbClearance()`. :+1: